### ISSUE TITLE
Tweak weighing algorithm? 

### ISSUE THREAD

**Issue by timtomch** [bgrsquared/places#5]
To reduce skewing produced by sparsely-populated areas.

On the Canadian map, some sparsely-populated areas with, say, a single place per dot will register at 100% if that place matches the search criteria. This drowns other denser areas where there might be matching places, say 100 out of 1000 for the same area size. This will register at 10% only even though there are more places here that match the criteria.

It looks like the code is already adjusting for that but I'm unsure how.

**Comment by chroth7** [comment]
Yes, there is indeed a very crude way of handling such cases in the code, let me try to explain how it works and maybe that solves the problem you are describing.

---

Per country, we have a parameter, presently called relevantNumber, see here:
https://github.com/bgrsquared/places/blob/master/app/config/globals.js#L166-L182

For larger countries, I just set it to 20 (going with Moritz Stefaner's example for Germany) and else to 5.

Next, when calculating the percentages per circle, I will on the fly calculate the maximum percentage I encounter to use the colors accordingly.
BUT to calculate the maxPercentage, I only take the countries into account that have more places than the relevantNumber indicates, see here:
https://github.com/bgrsquared/places/blob/master/app/reducers/helpers/dataStructure.js#L51-L53

In other words: if there are just 4 places in a node and all of them are selected, it doesn't change the maxPercentage if the relevantNumber is 5. (they will be dark, yes, but not affect the scale).

---

Do I make sense?

---

It is very tricky to set the relevantNumber.
Too high: rare patterns get boring colorscale (just yellow for 0 and dark for >0)
Too low: sparsely populated areas dominate the percentages

---

Summary:
- I don't think making this input dynamic in the GUI is smart, as it adds a lot of confusing and very little added value
- the current approach is limited, I think it's a good idea to improve it

Do I make sense? Let's discuss other options & thanks for the input
